sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-calculation-process.ts

```diff
@@ -136,4 +158,66 @@ export abstract class ECertCalculationProcess {
       }
     }
   }
+
+  /**
+   * Creates disbursement blocked notification for student.
+   * @param studentId student id associated with the blocked disbursement.
+   * @param entityManager entity manager to execute in transaction.
+   */
+  private async createDisbursementBlockedNotificationForStudent(
```

Comment:
I do not see a great benefit of using 2 private method here. But more than that, the same query is repeated twice. Hence I would advice one private method which creates 2 payloads and calls 2 methods in notification action service.

Reply:
Queries are different @dheepak-aot.

Reply:
As discussed, I meant repeating a similar query twice and a difference in selects of one or 2 properties is not a good enough reason to repeat the query.